.github/workflows/macos.yml

```diff
@@ -0,0 +1,269 @@
+          - -//google/cloud/talent/...
+          - -//google/cloud/retail/...
+          - -//google/cloud/monitoring/...
+          - -//google/cloud/appengine/...
+          - -//google/cloud/sql/...
+          - -//google/cloud/aiplatform/...
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ inputs.checkout-ref }}
+    - uses: google-github-actions/auth@v1
+      with:
+        create_credentials_file: true
+        credentials_json: ${{ secrets.BUILD_CACHE_KEY }}
+    - name: Install bash >= 5.x
+      run: brew install bash
+    - name: Build google-cloud-cpp
+      run: |
+        /usr/local/bin/bash ci/gha/builds/macos-bazel.sh ${{ join(matrix.targets, ' ') }}
+    env:
+      USE_BAZEL_VERSION: 6.2.1
+      BAZEL_REMOTE_CACHE: https://storage.googleapis.com/cloud-cpp-gha-cache/bazel-cache/${{ matrix.os }}
+
+  cmake:
+    name: cmake + ${{ matrix.os }} + ${{ matrix.shard }}
+    runs-on: ${{ matrix.os }}-xl
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    strategy:
+      # Continue other builds even if one fails
+      fail-fast: false
+      matrix:
+        os: [ macos-12 ]
+        shard: [ 'Core', 'Compute', 'Larger Libraries', 'Other' ]
+        include:
+        - shard: Core
+          features:
+          - bigtable
+          - pubsub
+          - spanner
+          - storage
+        - shard: Compute
+          features:
+          - compute
+        - shard: Larger Libraries
+          features:
+          # See above
+          - containeranalysis
+          - iam
+          - iap
+          - kms
+          - metastore
+          - notebooks
+          - run
+          - servicedirectory
+          - speech
+          - tpu
+          - trace
+          - vision
+          - binaryauthorization
+          - datacatalog
+          - dataplex
+          - gkemulticloud
+          - resourcemanager
+          - servicecontrol
+          - support
+          - contentwarehouse
+          - dataproc
+          - talent
+          - retail
+          - monitoring
+          - appengine
+          - sql
+          - aiplatform
+        - shard: Other
+          features:
+          - __ga_libraries__
+          - __experimental_libraries__
+          - -bigtable
+          - -pubsub
+          - -spanner
+          - -storage
+          - -compute
+          - -containeranalysis
+          - -iam
+          - -iap
+          - -kms
+          - -metastore
+          - -notebooks
+          - -run
+          - -servicedirectory
+          - -speech
+          - -tpu
+          - -trace
+          - -vision
+          - -binaryauthorization
+          - -datacatalog
+          - -dataplex
+          - -gkemulticloud
+          - -resourcemanager
+          - -servicecontrol
+          - -support
+          - -contentwarehouse
+          - -dataproc
+          - -talent
+          - -retail
+          - -monitoring
+          - -appengine
+          - -sql
```

Comment:
would experimental libraries be `experimental-sql` ?

Reply:
As you gathered, the CMake feature names are based on the directory name.